recipes/ifeature/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "ifeature" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a96534f3a9777d72531efce19030c59311216d9ed8366856b6eed7cf5bb258dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+
+  run:
+    - python
+    - six ==1.15.0
+    - numpy >=1.10.4
+    - pandas >=0.18.1
+    - tqdm ==4.49.0
+    - lightgbm ==3.0.0
+    - jieba
+    - wrapt ==1.12.1
+
+
+test:
+  imports:
+    - __utils
+    - ifeature
+    - ifeature.eda
+    - ifeature.features_add
+    - ifeature.features_sub
+    - ifeature.features_transform
+    - ifeature.models
+    - ifeature.models.utils
+    - ifeature.utils
+
+about:
+  home: "https://github.com/Jie-Yuan/iFeature"
+  license: MIT
```

Comment:
I don't see any license file in the repository. Can you please add one and point to it with the `license_file:` line?

Reply:
added

Reply:
The license file should go with your repository at https://github.com/Jie-Yuan/iFeature. Usually they are named "LICENSE"

Reply:
The owner of the Repo didn't add the License file but PyPi repo mentioned the MIT license.

Reply:
Ahh I see. So the license file usually is distributed with the source code. The [meta.yml docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file) specify that the `license_file` attribute should refer to the name of the license file in the source directory. Since there is not one, I think the `license_file` here can be left blank as you had it originally. Sorry for the confusion!

Reply:
Ok, thank you. I restore the original,  blank `license_file`.